src/components/profile/editprofile.jsx

```diff
@@ -0,0 +1,155 @@
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/no-unused-prop-types */
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/no-access-state-in-setstate */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import {
+  Header, Button, Form, Grid, Image,
+} from 'semantic-ui-react';
+
+import * as profileActions from '../../actions/profileAction';
+import '../../styles/profile.scss';
+
+class EditProfile extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Since redux is incorporated, consider handling state from the redux store.

Reply:
Thank you, will consider that in future